packages/preview/abyss-book/0.2.8/README.md

```diff
@@ -0,0 +1,77 @@
+# AbyssBook
+
+一个黑色Typst模板
+
+# 安装
+
+## macOS
+
+`git clone https://github.com/CrossDark/AbyssBook.git "$HOME/Library/Application Support/typst/packages/local/abyss-book/0.2.8"`
+
+# 使用
+
+```typst
+#import "@local/abyss-book:0.2.8": *
```

Comment:
The recommended way to use the package should be to use the `@preview` import, so that the compiler can automatically download the package both from the web-app and the CLI. It is fine to document how to install it manually, but it should not be the main way.